configuration/configuration_blackbox_test.go

```diff
@@ -141,3 +150,49 @@ func TestGetKeycloakEndpointUserInfoSetByEnvVaribaleOK(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "userinfoEndpoint", url)
 }
+
+func TestGetTokenPrivateKeyFromConfigFile(t *testing.T) {
+	t.Parallel()
+
+	envKey := generateEnvKey(varTokenPrivateKey)
+
+	realEnvValue := os.Getenv(envKey) // could be "" as well.
+
+	os.Unsetenv(envKey)
+	defer os.Setenv(envKey, realEnvValue) // set it back before we leave.
```

Comment:
When you overwrite environment variables, the test cannot be marked to be executed in parallel. Please remove `t.Parallel()`.